stock_picking_package_preparation_line/README.rst

```diff
@@ -0,0 +1,39 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
+Stock Picking Package Preparation
+=================================
+
+This module adds a new class on **Package Preparation** to manage details
+related with stock move or not
+
+Installation
+============
+
+ * This module depends only on the **stock_picking_package_preparation** module.
+
+Configuration
+=============
+
+ * No special setup
+
```

Comment:
Please add **Usage** (with only **try on runbot**  button) and **Bug Tracker** sections.